### ISSUE TITLE
Test: [alert] fix unstable tests

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-components#1608]
### Summary

The [`a11y tests`](https://github.com/Esri/calcite-components/blob/02be9a4/src/components/calcite-alert/calcite-alert.e2e.ts#L13-L25) tests fail intermittently, which bogs down builds and deployments. 

These tests should be restored when they are stable and reliable.

**Comment by jcfranco** [comment]
Closing as self-verified based on latest Travis builds. cc @benelan
